config/webpack.config.js

```diff
@@ -144,9 +143,7 @@ module.exports = function(webpackEnv) {
 	};
 
 	const entry = Object.entries(entries).reduce((newObj, [entry, path]) => {
-		newObj[entry] = [isEnvDevelopment && require.resolve('react-dev-utils/webpackHotDevClient'), path].filter(
-			Boolean
-		);
```

Comment:
Since we are not using HMR, we don't need `'react-dev-utils/webpackHotDevClient'`